project-contribution.md

```diff
@@ -0,0 +1,4 @@
+## Project I wish to contribute to:
```

Comment:
pick a project and put in the appropriate `md` file in the appropriate folder, e.g. `projects/python/numpy.md`